docs/source/usersguide/depletion.rst

```diff
@@ -128,6 +128,9 @@ For our example problem, this would deplete fuel on the outer region of the prob
 with different reaction rates than those in the center. Materials will be depleted
 corresponding to their local neutron spectra, and have unique compositions at each
 transport step.
+The volume of the original ``fuel_3`` material must represent the volume of
```

Comment:
Can you join this with the previous line so the flow just continues?

Reply:
Sure thing! Made that whole block more condensed